Setup/docker-compose_dev.yml

```diff
@@ -140,9 +143,12 @@ services:
         - "CONSUL_BIND_INTERFACE=eth0"
         - "NUMBER_OF_EXPECTED=3" # works for single and multi node
         - "LOGSTASH_FIELDS=service=consulserver"
+        - "DATA_DIR={{.Service.Name}}{{.Task.Slot}}" #single node don't use this for multinode
```

Comment:
What should be done for multinode? just comment this line?

Reply:
Yes.
In multinode each volume on separate node